### PR TITLE
Changed some extrinsics to DispatchType::Operational

### PR DESCRIPTION
**Pull request by syan095**
# Pull Request

Closes: PRO-913

## Checklist

Please conduct a thorough self-review before opening the PR.

- [x] I am confident that the code works.
- [x] I have updated documentation where appropriate.

## Summary

Changed some Validator extrinsic to DispatchClass::Operational
Adjusted DispatchClass::Normal weight (from 75%) to 25% in the Runtime

### CODE REVIEW

**Comment by dandanlen** [comment]
After reviewing some real-world weights on mainnet I opted to set the weight proportion to 50-50. Currently the ratio of weights fluctuates between ~ 1:1 and 3:1 operational to normal. But we're also only at ~14% max block consumption. 

The main update here is tagging the extrinsics as operational - we can adjust the weight targets another time if deemed necessary.

**Review by kylezs — APPROVED** [review]

**Comment by dandanlen** [comment]
Sorry - bad merge caused the whole org to be tagged for review 😓

File: state-chain/pallets/cf-witnesser/src/lib.rs [hunk 1]
```diff
@@ -376,10 +376,10 @@ pub mod pallet {
 		/// - [DuplicateWitness](Error::DuplicateWitness)
 		#[allow(clippy::boxed_local)]
 		#[pallet::call_index(0)]
-		#[pallet::weight(
+		#[pallet::weight((
 			T::WeightInfo::witness_at_epoch().saturating_add(call.get_dispatch_info().weight /
 				T::EpochInfo::authority_count_at_epoch(*epoch_index).unwrap_or(1u32) as u64)
-		)]
+		, DispatchClass::Operational))]
 		pub fn witness_at_epoch(
```
Comment (kylezs) [inline state-chain/pallets/cf-witnesser/src/lib.rs:383]: force_witness too?

**Review by kylezs — APPROVED** [review]

File: state-chain/pallets/cf-witnesser/src/lib.rs (hunk 1 above)
Comment (dandanlen) [reply state-chain/pallets/cf-witnesser/src/lib.rs:383]: Force witness is a governance extrinsic so it's only ever dispatched via goverannce approval.

**Pull request merged**